.github/workflows/check-milestone.yml

```diff
@@ -0,0 +1,80 @@
+name: Milestone Reminder
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - "master"
+
+jobs:
+  milestone-reminder:
+    if: github.event.pull_request.merged == 'true'
+    name: Remind to set milestone
+    runs-on: ubuntu-22.04
+    timeout-minutes: 5
+    permissions:
+      pull-requests: write
+      issues: read
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          script: | # js
```

Comment:
https://marketplace.visualstudio.com/items?itemName=limejuly.vscode-yaml-block-highlight is a game changer for writing JS in github actions. simple syntax highlighting saves so many errors.